src/test/kotlin/com/terraformation/backend/seedbank/model/accession/AccessionModelTests.kt

```diff
@@ -0,0 +1,174 @@
+package com.terraformation.backend.seedbank.model.accession
+
+import com.terraformation.backend.db.default_schema.UserId
+import com.terraformation.backend.db.seedbank.AccessionId
+import com.terraformation.backend.db.seedbank.AccessionState
+import com.terraformation.backend.db.seedbank.DataSource
+import com.terraformation.backend.db.seedbank.ProcessingMethod
+import com.terraformation.backend.db.seedbank.SeedQuantityUnits
+import com.terraformation.backend.db.seedbank.ViabilityTestId
+import com.terraformation.backend.db.seedbank.ViabilityTestResultId
+import com.terraformation.backend.db.seedbank.ViabilityTestSubstrate
+import com.terraformation.backend.db.seedbank.ViabilityTestType
+import com.terraformation.backend.db.seedbank.WithdrawalId
+import com.terraformation.backend.db.seedbank.WithdrawalPurpose
+import com.terraformation.backend.seedbank.grams
+import com.terraformation.backend.seedbank.model.AccessionModel
+import com.terraformation.backend.seedbank.model.SeedQuantityModel
+import com.terraformation.backend.seedbank.model.ViabilityTestModel
+import com.terraformation.backend.seedbank.model.ViabilityTestResultModel
+import com.terraformation.backend.seedbank.model.WithdrawalModel
+import com.terraformation.backend.seedbank.seeds
+import java.time.Clock
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneOffset
+
+abstract class AccessionModelTests {
+  val today = january(2)
```

Comment:
Very minor nitpick, but easy to change: Everything in here could have `protected` visibility.

Reply:
can do! I also changed the visibilities of the new classes to `internal` which I'd neglected previously